tools/test.py

```diff
@@ -87,17 +87,20 @@ def parse_args():
         action=DictAction,
         help='custom options for show_result. key-value pair in xxx=yyy.'
         'Check available options in `model.show_result`.')
+    parser.add_argument(
+        '--device', default=None, help='device used for testing. (Deprecated)')
```

Comment:
Why not keep this interface?
I prefer the use ```xxx --device cpu```  rather than ```export CUDA_VISIBLE_DEVICES=-1  xxxx```.

the command is simpler.

Reply:
To use `MMDataParallel` for CPU training/test, we need to specify `CUDA_VISILBE_DEVICES=-1`, and in that situation, the `--device` argument is useless.